termite.cc

```diff
@@ -1648,12 +1663,16 @@ static void set_config(GtkWindow *window, VteTerminal *vte, GtkWidget *scrollbar
     }
 
     if (info->clickable_url) {
-        info->tag = vte_terminal_match_add_regex(vte,
-                vte_regex_new_for_match(url_regex,
-                                        (gssize) strlen(url_regex),
-                                        PCRE2_MULTILINE | PCRE2_NOTEMPTY,
-                                        nullptr),
+        n_url_regexes = G_N_ELEMENTS (url_regex_patterns);
+        for (int i = 0; i < n_url_regexes; ++i)
+        {
```

Comment:
Just use `G_N_ELEMENTS()` directly in the loop condition, the compiler is smart enough to figure out that the macro expands to a constant value:

```suggestion
        for (int i = 0; i < G_N_ELEMENTS(url_regex_patterns); ++i) {
```

Reply:
yes, thanks.